apps/main-landing/src/app/creators/donate/page.tsx

```diff
@@ -13,7 +13,7 @@ export default function Donors() {
   return (
     <Providers>
       <TopBar />
-      <main className="relative flex min-h-screen grow flex-col items-center justify-around gap-4 overflow-hidden bg-[radial-gradient(111.94%_122.93%_at_16.62%_0%,_#E7F5E7_0%,_#b5d8ae_100%)] px-2 pb-1 pt-[56px] lg:flex-row lg:items-start lg:justify-center lg:px-0">
+      <main className="relative flex min-h-screen grow flex-col items-center justify-around gap-4 overflow-hidden bg-[radial-gradient(111.94%_122.93%_at_16.62%_0%,_#E7F5E7_0%,_#76C282_100%)] px-2 pb-1 pt-[56px] lg:flex-row lg:items-start lg:justify-center lg:px-0">
```

Comment:
It doesn't look like the color on the design:
![image](https://github.com/user-attachments/assets/0f441fbe-0228-481d-8e7b-708b6d0dbd0f)